lint/lint_test.sh

```diff
@@ -12,10 +12,21 @@ source "${RUNFILES_DIR:-/dev/null}/$f" 2>/dev/null || \
   { echo>&2 "ERROR: cannot find $f"; exit 1; }; f=; set -e
 # --- end runfiles.bash initialization v3 ---
 
-for report in {{reports}}; do
-  report_file=$(rlocation "$report")
+function assert_diagnostics_empty() {
+  report_file=$(rlocation "$1")
   if [ -s "${report_file}" ]; then      
         cat ${report_file}
         exit 1
   fi
-done
\ No newline at end of file
+}
+
+function assert_exit_code_zero() {
+  exit_code=$(cat $(rlocation "$1"))
+  diagnostics_file=$(rlocation "$2")
```

Comment:
is this the `report_file`? "diagnostics" is a term not used anywhere else

Reply:
changed "diagnostics" to "output" in the case where a report output group could be confused with a report file in that output group